src/main/res/values/strings.xml

```diff
@@ -680,18 +680,18 @@
     <string name="album_already_first_image">Already at first image in album.</string>
     <string name="album_already_last_image">Already at last image in album.</string>
 
-    <string name="upgrade_v190_login_message">RedReader has upgraded, and now uses oAuth for reddit authentication. If you were signed in to an account before the upgrade, you are now logged out.</string>
+    <string name="upgrade_v190_login_message">This new version of RedReader uses OAuth to log in on Reddit. Any account in use before the upgrade is now logged out.</string>
```

Comment:
This deliberately mentioned upgrades as it's only shown after upgrading, not on new installs. I'd probably just leave this string alone as I doubt it's been shown to anyone in years :smile:

Reply:
If it is in there, it should be consistent. I can comment it out if you want?

Reply:
It's still used by the code, but will only appear in the very specific case that someone upgrades from v1.8.7.8 (from several years ago) to a newer version.

Every second spent worrying about this string is a second wasted, so let's just leave it as it was :smile:

Reply:
There is also "reddit" and the double "you". "signed in" isn't used elsewhere. It still says upgrade. Not twice now, which helps the user figure out what an upgrade even is. That is better.